src/radical/pilot/agent/lm/base.py

```diff
@@ -167,9 +169,11 @@ def create(cls, name, cfg, session):
             return impl(name, cfg, session)
 
         except KeyError:
+            # pylint: disable=protected-access
             session._log.exception("LM '%s' unknown or defunct" % name)
 
         except Exception as e:
+            # pylint: disable=protected-access
```

Comment:
This should use `self._log` as defined in the ctor instead of adding the pyling exception.